frontend/src/routes/Infrastructure/Clusters/DiscoveredClusters/DiscoveryConfig/DiscoveryConfig.tsx

```diff
@@ -81,6 +85,46 @@ export default function DiscoveryConfigPage() {
     >
       <AcmPageContent id="discoveryConfigPageContent">
         <PageSection variant="light">
+          <Alert
+            variant="info"
+            isInline
+            title="Import your ROSA clusters automatically"
+            style={{ marginBottom: '1em' }}
+            className="pf-v5-u-mb-md"
+            actionLinks={[
+              [
+                <AlertActionLink
```

Comment:
@Randy424 Sorry - I didn't mean to suggest you should use 1 `AlertActionLink` and 1 `Button`... just that either would work.

Reply:
You can remove the `variant="link"` and `isInline` properties - AlertActionLink is just a Button with those properties set already.